tools/support_matrix/lib/support_matrix.rb

```diff
@@ -17,8 +17,9 @@ def write(path)
 
   def merge_source(name)
     result = {}
+    merger = proc { |key,v1,v2| Hash === v1 && Hash === v2 ? v1.merge(v2, &merger) : v2 }
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.<br>Avoid the use of the case equality operator `===`.<br>Space missing after comma.